db/migrations/0001_initial.py

```diff
@@ -40,19 +71,44 @@
             name='Movie',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('title', models.CharField(max_length=255)),
+                ('title', models.CharField(db_index=True, max_length=255)),
                 ('description', models.TextField()),
-                ('actors', models.ManyToManyField(to='db.Actor')),
-                ('genres', models.ManyToManyField(to='db.Genre')),
+                ('actors', models.ManyToManyField(related_name='movies', to='db.Actor')),
+                ('genres', models.ManyToManyField(related_name='movies', to='db.Genre')),
             ],
         ),
         migrations.CreateModel(
             name='MovieSession',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('show_time', models.DateTimeField()),
-                ('cinema_hall', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.cinemahall')),
-                ('movie', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.movie')),
+                ('cinema_hall', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.cinemahall')),
+                ('movie', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.movie')),
             ],
         ),
+        migrations.CreateModel(
+            name='Order',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='orders', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'ordering': ['-created_at'],
+            },
+        ),
+        migrations.CreateModel(
+            name='Ticket',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('row', models.IntegerField()),
+                ('seat', models.IntegerField()),
+                ('movie_session', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.moviesession')),
+                ('order', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='tickets', to='db.order')),
+            ],
+        ),
+        migrations.AddConstraint(
+            model_name='ticket',
+            constraint=models.UniqueConstraint(fields=('row', 'seat'), name='unique_row_and_seat'),
```

Comment:
The `UniqueConstraint` on the `Ticket` model should include the `movie_session` field to ensure that the combination of `row`, `seat`, and `movie_session` is unique. This will prevent duplicate tickets for the same seat in the same session.